### PR TITLE
Write new block to the local directory.

### PR DESCRIPTION
**Pull request by rahulgi**
*dependent on #7. land that first.*

### CODE REVIEW

File: src/createBlock.ts [hunk 1]
```diff
@@ -138,6 +151,6 @@ async function createBlock(): Promise<Block> {
 }
 
 createBlock().then((block) => {
-    console.log(formatData(block))
+    // console.log(formatData(block))
```
Comment (markulrich) [inline src/createBlock.ts]: ?

File: src/createBlock.ts [hunk 2]
```diff
@@ -118,6 +121,15 @@ function formatData(data) {
     return JSON.stringify(data, null, 4);
 }
 
+/**
+ * Write the block to a file.
+ */
+function writeBlockDataToLocalFile(blockData, multiHash, formattedData) {
+    const filename = `block-${blockData.sequence}-${multiHash}.json`;
+    fs.writeFileSync(filename, formattedData);
```
Comment (markulrich) [inline src/createBlock.ts]: Best practice probably to use fs.writeFile and make this function async, though that would require figuring out `util.promisify` which only works in certain versions of Node. Maybe just put a TODO?

**Review by markulrich — APPROVED** [review]

File: src/createBlock.ts (hunk 2 above)
Comment (osdiab) [reply src/createBlock.ts]: If you want it to be cross platform you can use `bluebird.promisify` and add `bluebird` as a dep

File: src/createBlock.ts [hunk 3]
```diff
@@ -127,6 +139,7 @@ async function createBlock(): Promise<Block> {
     const data = await createIpfsBlock();
     const formattedData = formatData(data);
     const multiHash = await saveDataToIpfsAsFile(`block-${data.sequence}`, formatData);
+    writeBlockDataToLocalFile(data, multiHash, formattedData);
```
Comment (osdiab) [inline src/createBlock.ts]: This probably should be async, and also all of this probably should have some error handling logic, given that writing to both IPFS and disk can go wrong.

**Review by osdiab — DISMISSED** [review]

File: src/createBlock.ts (hunk 3 above)
Comment (rahulgi) [reply src/createBlock.ts]: i just dropped the whole function in a try-catch. is that wrong? :P

File: src/createBlock.ts (hunk 2 above)
Comment (rahulgi) [reply src/createBlock.ts]: I manually created and returned a promise. Is it better to use promisify?

**Pull request merged**